quickwit-indexing/src/actors/indexing_server.rs

```diff
@@ -544,6 +544,29 @@ mod tests {
         client.spawn_pipelines(index_id.clone()).await.unwrap();
         assert_eq!(client.observe_server().await.num_running_pipelines, 2);
 
+        // Test `spawn_merge_pipeline`.
+        let merge_pipeline_id = client
+            .spawn_merge_pipeline(index_id.clone(), true, false)
+            .await
+            .unwrap();
+        assert_eq!(client.observe_server().await.num_running_pipelines, 3);
+        assert_eq!(
```

Comment:
`.observe_pipeline` is fairly slow. Let's store the observation in one variable and perform the two asserts on the same observation.